challenge.md

```diff
@@ -6,19 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+git clone + SSH URL
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+git log
```

Comment:
Nope - this should be `git status`. `git status` will show a list of all the files that have been created/modified since the last commit. Additionally, you can also use `git diff` to look at the specific changes.